components/recruit_crm/actions/create-cadidate/create-cadidate.mjs

```diff
@@ -0,0 +1,90 @@
+import app from "../../recruit_crm.app.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "recruit_crm-create-cadidate",
+  name: "Create Cadidate",
```

Comment:
A couple typos. Also the filename and foldername should be `create-candidate.mjs` and `create-candidate`.
```suggestion
  key: "recruit_crm-create-candidate",
  name: "Create Candidate",
```